src/hugo/content/en-us/implementations/aws/command/command2/_index.md

```diff
@@ -0,0 +1,721 @@
+device -> broker : subscribe("$aws/things/device1/shadow/get/rejected")
+device -> broker : subscribe("$aws/things/device1/shadow/update/accepted")
+device -> broker : subscribe("$aws/things/device1/shadow/update/rejected")
+device -> broker : subscribe("$aws/things/device1/shadow/update/delta")
+
+== Application connect and subscribe ==
+app -> broker : connect(iot_endpoint)
+app -> broker : subscribe("$aws/things/device1/shadow/update/accepted")
+app -> broker : subscribe("$aws/things/device1/shadow/update/rejected")
+
+== App to Device state control and update ==
+app -> broker : publish("$aws/things/device1/shadow/update", "desired : {motorStatus: on}")
+broker -> shadow : updates shadow for desired state changes from app
+device -> broker : On initial connect/reconnect, publish("$aws/things/device1/shadow/get")
+broker -> device : Send persisted shadow state, publish("$aws/things/device1/shadow/get/accepted") 
+device -> device : Turn motor on
+device -> broker : publish("$aws/things/device1/shadow/update", "reported : {motorStatus: on}")
+broker -> shadow : updates reported state on shadow file
+broker -> app : publish("$aws/things/device1/shadow/update/accepted", "success")
+app -> app : reconcile("device1")
+rnote over device #FFFFFF: If device remained connected
+app -> broker : publish("$aws/things/device1/shadow/update", "desired : {motorStatus: off}")
+broker -> shadow : updates shadow for desired state changes from app
+broker -> device : publish("$aws/things/device1/shadow/update/delta", "desired : {motorStatus: off}")
+device -> device : Turn motor off
+device -> broker : publish("$aws/things/device1/shadow/update", "reported : {motorStatus: off}")
+broker -> shadow : updates reported state on shadow file
+broker -> app : publish("$aws/things/device1/shadow/update/accepted", "success")
+app -> app : reconcile("device1")
+
+
+@enduml
+```
+
+
+
+{{% /tab %}}
+{{% tab name="Device Action" %}}
+
+
+![Command and control device action using shadow](architecture2.svg)
+
+- _AWS IoT Core_ is the MQTT message broker processing messages
+- _Device_ is the IoT thing to be controlled
+- _Client Application_ is the remote logic that issues commands, updates device configuration and consumes device telemetry data
+- _Device Shadow_ is the replica which makes device state available to applications and other services
+- _ShadowTopicPrefix_ is an abbreviated form of the topic, referred as `$aws/things/device1/shadow` (classic shadow)
+
+1. The _Device_ establishes an MQTT connection to the _AWS IoT Core_ endpoint and then subscribes to the reserved [MQTT shadow topics](https://docs.aws.amazon.com/iot/latest/developerguide/reserved-topics.html#reserved-topics-shadow) to get and update shadow event operations. Moreover, instead of subscribing individually to each reserved shadow topic, wildcard(_#_) could be used for generalized subscription like _`ShadowTopicPrefix/get/#`_ , _`ShadowTopicPrefix/update/#`_.
```

Comment:
"a wildcard(_#_)"

Same comment on the other tab for dealing with the listing of the topics.